python/gui/auto_generated/qgsattributeform.sip.in

```diff
@@ -44,6 +44,14 @@ class QgsAttributeForm : QWidget
 
     const QgsFeature &feature();
 
+    const QgsFeature &currentFormFeature();
```

Comment:
```suggestion
    QgsFeature currentFormFeature() const;
```

Better safe than sorry, returning refs has potential risks if used unconsciously and returning an implicitly shared construct like a feature is almost free.

Reply:
Okay.